io.github.Qalculate.yaml

```diff
@@ -0,0 +1,107 @@
+app-id: io.github.Qalculate
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: qalculate-gtk
+rename-appdata-file: qalculate-gtk.appdata.xml
+rename-desktop-file: qalculate-gtk.desktop
```

Comment:
It would be nice to rename those files upstream to avoid having two identical apps that store their reviews on GNOME 
Software as two different apps. Make sure to you use the `<provides><id>$oldID</id></provides>` on your appdata if you do so. It will tell the Software centers out there that it's the same app under a new app-id. This is not a blocker but rather a suggestion

Reply:
I will take that under consideration for the next version of Qalculate.